MagneticField/ParametrizedEngine/BuildFile.xml

```diff
@@ -2,6 +2,8 @@
 <use name="FWCore/ParameterSet"/>
 <use name="MagneticField/Engine"/>
 <use name="MagneticField/UniformEngine"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
+<flags ALPAKA_BACKENDS="1"/>
```

Comment:
@smuzaffar do we need to add this flag, if the only alpaka files are under `interface/alpaka` ?
That is, there are no files in `src/alpaka`.

Reply:
No, there is no need to add this flag in such case. SCRAM only generates alpaka backend products if there are source files under src/alpaka.

Reply:
OK, thanks.

@lecriste can you remove this flag ?
```suggestion
```

Reply:
Removed: https://github.com/cms-sw/cmssw/pull/44360/commits/a088e4686ce71a97ce88a7c87293e4d05854fda8